docs/xrpl-wallet/api-reference/xrpl/sign-message.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: Sign Message
+---
+
+To sign a message, you can use the `signMessage` method.
+
+---
+
+`signMessage`
+
+### Parameters
+
+`params`
+
+| Name     | Type                | Description             |
+| -------- | ------------------- | ----------------------- |
+| `params` | `SignMessageParams` | Sign Message Parameters |
+
+`SignMessageParams`
+
+| Name      | Type     | Description                  |
+| --------- | -------- | ---------------------------- |
+| `message` | `string` | Message to sign              |
+| `from`    | `string` | Address to sign message with |
+
+### Reference
+
+```typescript
+async signMessage(params: SignMessageParams): Promise<{ signature: string }> {
+    if (!this.isInitialized) throw new Error("Torus is not initialized");
```

Comment:
add a comment to tell what should be value of "this" here or may be initialize torus before starting this example on top.

Reply:
Also we are providing an example below, i think we don't need this reference.